### PR TITLE
Stop Reloading

### PR DESCRIPTION
**Pull request by AshNewar**

### CODE REVIEW

File: .eslintrc.json
Comment (abhishekshree) [inline .eslintrc.json:1]: don't change the lint without a discussion

File: next.config.js
Comment (abhishekshree) [inline next.config.js:1]: unnecessary

**Review by abhishekshree — CHANGES_REQUESTED** [review]
Good work, just don't change configs on your own bacche

**Comment by utkarsh-tf141** [comment]
@AshNewar Resolve merge conflicts and the changes requested

**Pull request closed without merging**